README.md

```diff
@@ -1,60 +1,88 @@
 CakePHP Sample App on OpenShift
------------------
+===============================
 
-This example will serve an http response of various "machine" stats from the "machine" the CakePHP app is running on to [http://host:8080](http://host:8080).
+This is a quickstart CakePHP application for OpenShift v3 that you can use as a starting point to develop your own application and deploy it on an [OpenShift](https://github.com/openshift/origin) cluster.
 
-### OpenShift setup ###
+If you'd like to install it, follow [these directions](https://github.com/openshift/cakephp-ex/blob/master/README.md#installation).  
 
-One possible option is to use the Docker all-in-one launch as described in the [origins project](https://github.com/openshift/origins).
+The steps in this document assume that you have access to an OpenShift deployment that you can deploy applications on.
 
-### The project ###
+OpenShift Considerations
+------------------------
+These are some special considerations you may need to keep in mind when running your application on OpenShift.
 
-If you don't have a project setup all ready, go ahead and take care of that
+###Security
+Since these quickstarts are shared code, we had to take special consideration to ensure that security related configuration variables was unique across applications. To accomplish this, we modified some of the configuration files. Now instead of using the same default values, OpenShift can generate these values using the generate from logic defined within the instant application's template.
 
-        $ oc new-project cakephp --display-name="CakePHP" --description="Sample CakePHP app"
+OpenShift stores these generated values in configuration files that only exist for your deployed application and not in your code anywhere. Each of them will be unique so initialize_secret(:a) will differ from initialize_secret(:b) but they will also be consistent, so any time your application uses them (even across reboots), you know they will be the same.
 
-That's it, project has been created.  Though it would probably be good to set your current project to this (thought new-project does it automatically as well), such as:
+TLDR: OpenShift can generate and expose environment variables to your application automatically. Look at this quickstart for an example.
 
-        $ oc project cakephp
+###Installation: 
+These steps assume your OpenShift deployment has the default set of ImageStreams defined.  Instructions for installing the default ImageStreams are available [here](http://docs.openshift.org/latest/admin_guide/install/first_steps.html)
 
-### The app ###
+1. Fork a copy of [cakephp-ex](https://github.com/openshift/cakephp-ex)
+2. Clone your repository to your development machine and cd to the repository directory
+3. Add a PHP application from the provided template and specify the source url to be your forked repo  
```

Comment:
since the instructions for the command to run are immediately below this statement, i don't think it needs additional clarification.